### ISSUE TITLE
Samsung galaxy SII - front camera switch not work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#326]
```
really great program, only one who work great with local PBX for video IP, but 
still one thing missing for me -
Samsung galaxy SII - front camera switch not work

```

Original issue reported on code.google.com by `vil...@gmail.com` on 6 Feb 2012 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
fixed in 2.0.453
```

Original comment by `vil...@gmail.com` on 6 Feb 2012 at 7:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 17 Feb 2012 at 12:11
- Changed state: **Invalid**
